### DISCUSSION TITLE
Add `exclude_computed_fields` to model dump

### DISCUSSION THREAD

**Discussion by 1298se** [pydantic/pydantic#10625] [Feature Request]
When marshalling a Pydantic model for database storage we don't need to store computed fields. It would be nice if there was an argument to support this.

**Comment by Viicos** [comment]
See https://github.com/pydantic/pydantic/discussions/7114

**Comment by Viicos** [comment]
A `exclude_computed_fields` option is added in https://github.com/pydantic/pydantic/pull/12334, landing in 2.12.

**However**, please consider using `round_trip` if you are trying to achieve round-tripping behavior!
